.github/workflows/stale.yml

```diff
@@ -13,7 +13,11 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         days-before-stale: 60
         days-before-close: 7
-        stale-issue-message: 'This issue has been marked as stale because it has been open for a while and has had no recent activity. If this issue is still important to you please drop a comment below and we will add this to our backlog to complete. Otherwise, it will be closed in 7 days.'
-        stale-issue-label: 'stale'
-        stale-pr-message: 'This PR has been marked as stale because it has been open for a while and has had no recent activity. If this PR is still important to you please drop a comment below and we will add this to our backlog to complete. Otherwise, it will be closed in 7 days.'
-        stale-pr-label: 'stale'
+        stale-issue-message: 'Hi, this issue has been open for a while and has had no recent activity so it has been labeled for attention. With significant change within our team over the past 6 months, we are no longer aware if your issue is still relevant and/or up-to-date.
+                              
+                              If this issue is still important and relevant to you, please drop a comment below and we will add this back to our community workload for prioritisation. Otherwise, it will be considered inactive and will be closed after 7 days. Closed issues can still be reopened if you comment on them shortly after it has been closed.'
```

Comment:
What about if we add a line that says something like

`If you are waiting on us for a response, we will pick it up in the triage` 

or something along the lines of that..

Reply:
What do you think about the days until closed? I think maybe we could allow for more time now.. maybe 30 days?